pkg/event/event.go

```diff
@@ -23,11 +23,16 @@ import (
 	"github.com/redhat-cne/sdk-go/pkg/types"
 )
 
+const (
+	// EVENT is the generic event type
+	EVENT = "event"
```

Comment:
do we need this?

Reply:
created redfish/types.go to replace this. The default eventType for redfish event is "Alert" and is the only type we support for now.

Reply:
done

Reply:
but you have not deleted const EVENT here?

Reply:
done. deleted this generic EVENT type.